### ISSUE TITLE
What's the best way to see release updates?

### ISSUE THREAD

**Issue by WeirdBeardDev** [microsoft/Microsoft.Unity.Analyzers#152]
I noticed the project recently release v1.10.0, however, I don't see a release here or any release notes on NuGet.  What's the best way to find out what changed between releases?

**Comment by sailro** [comment]
Hi @WeirdBeardDev 

That's a very good question.

For now, given we ship those analyzers along with [VSTU](https://docs.microsoft.com/en-us/visualstudio/gamedev/unity/change-log-visual-studio-tools-for-unity) we do not have a real changelog for this repo.

v1.10.0 will be shipped with VSTU 4.10.0.0. But the changelog will not be available until VSTU is fully published. (latest stable version is 4.8.2.0 so with analyzers v1.8.2)

As a workaround, I pushed version-tags so you can see at least the docs to quickly see changes:

Changes between 1.9.1 and 1.10.0:
https://github.com/microsoft/Microsoft.Unity.Analyzers/compare/1.9.1...1.10.0?short_path=47c5f70#diff-47c5f70f455a0039334dd042f30fbb585c977a73ad838ee4747e5cd304ffb412

To summarize:

We added [UNT0019](https://github.com/microsoft/Microsoft.Unity.Analyzers/blob/main/doc/UNT0019.md), [UNT0020](https://github.com/microsoft/Microsoft.Unity.Analyzers/blob/main/doc/UNT0020.md), [UNT0021](https://github.com/microsoft/Microsoft.Unity.Analyzers/blob/main/doc/UNT0021.md), [UNT0022](https://github.com/microsoft/Microsoft.Unity.Analyzers/blob/main/doc/UNT0022.md), [UNT0023](https://github.com/microsoft/Microsoft.Unity.Analyzers/blob/main/doc/UNT0023.md) diagnostics, and [USP0017](https://github.com/microsoft/Microsoft.Unity.Analyzers/blob/main/doc/USP0017.md) suppressor.

Thanks!

**Comment by WeirdBeardDev** [comment]
That's great, I'll read over those.  I use VSCode so I integrate the DLL directly in, as I don't think VSTU would work in VSCode.  TBH I'm not sure about that.

**Comment by sailro** [comment]
I started https://github.com/microsoft/Microsoft.Unity.Analyzers/releases to track the changelog here.
